swagger/swagger-invocation/invocation-core/src/main/java/org/apache/servicecomb/swagger/invocation/exception/ExceptionFactory.java

```diff
@@ -32,7 +32,7 @@
   //    private static final int PROVIDER_INNER_STATUS_CODE = 510;
   public static final int PRODUCER_INNER_STATUS_CODE = 590;
 
-  public static final String PRODUCER_INNER_REASON_PHRASE = "Cse Internal Server Error";
+  public static final String PRODUCER_INNER_REASON_PHRASE = "Unexpected error happened, please check your server logic and error log";
```

Comment:
`Unexpected producer error happened, please check logs for details.`
`Unexpected consumer error happened, please check logs for details.`

Reply:
ok